src/models/configuration.ts

```diff
@@ -1,5 +1,10 @@
 // Libs
-import { types, Instance } from "mobx-state-tree";
+import { types, Instance, getRoot } from "mobx-state-tree";
+import { flow } from "mobx";
```

Comment:
sauf erreur, tu as aussi intérêt à utiliser flow de MST (je me suis battu avec un bug une fois qui était du à ça, mais c'est peut-être plus un problème)

Reply:
Updated in PR #11